src/main/scala/Main.scala

```diff
@@ -42,7 +42,7 @@ object Main {
       analysisResultsDot: Option[String]
   )
 
-  def main(args: Iterable[String]): Unit = {
+  def main(args: Array[String]): Unit = {
```

Comment:
This is just a reversion isn't it? So why does it break BitVectorAnalysisTests? I'm a bit baffled